doc/release-notes/2025-03-26.rst

```diff
@@ -0,0 +1,63 @@
+.. _release_notes_2025_03_26:
+
+Highlights from 2025-03-26 release
+==================================
+
+Enhancements
+------------
+
+- Jakub continues on adding support for Pulp in Copr. We are slowly adding more
+  features to it. Most notable additions are:
+  - Regenerate devel repositories `PR#3524`_.
+  - Implement CDN baseurl `PR#3657`_.
+  - Implement backward compatibility for old repofiles `PR#3504`_.
+  - And more: `PR#3509`_, `PR#3461`_.
+- Copr has integrated `log_detective`_ service for failed builds directly to the UI.
+  Try it out by clicking the Ask AI link about failure explanation. If you want to help us
+  with creating better model, you can click Tech AI and provide a feedback about your failed
+  build. The AI service is still in beta, so some bugs might occur. `PR#3608`_.
+- Custom Webhook now adds a record in Webhook History table. `PR#3456`_.
+- Add footer note thanking and mentioning who sponsors the Fedora Copr project `PR#3600`_.
+- Introduce OIDC authentication for Copr. `PR#3642`_.
+
+
+Bugfixes
+--------
+
+- Fix safe.directory error for make_srpm method. `PR#3464`_.
+- Fix unexpected 400 errors `PR#3472`_.
+- Fix "Expire now" button in the chroot page. `PR#3599`_.
+- Fix "dnf4 copr enable" traceback on non-existing projects. `PR#3594`_.
+- /build/source-chroot endpoint dissapeared for a while, but it is back now. `PR#3578`_.
+- Limit for builds per architecture was mistakenly applied to all architectures `PR#3575`_.
+- Fix log subject in emails informing about builds. `PR#3530`_.
+- Skip only if all macros in ExclusiveArch and ExcludeArch evaluated. `PR#3513`_.
+- Fix bad formating when non-existing project was requested. `PR#3651`_.
```

Comment:
fixed custom method  - builder machines no longer assume `/bin/su` works in minimal Mock chroot (it doesn't in Fedora 42), #3631